app/views/transaction_categories/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_with(model: transaction_category) do |form| %>
+  <% if transaction_category.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(transaction_category.errors.count, "error") %> prohibited this transaction_category from being saved:</h2>
+
+      <ul>
+        <% transaction_category.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :category_id, style: "display: block" %>
+    <%= form.text_field :category_id %>
+  </div>
+
+  <div>
+    <%= form.label :transaction_id, style: "display: block" %>
+    <%= form.text_field :transaction_id %>
+  </div>
+
+  <div>
+    <%= form.submit %>
+  </div>
```

Comment:
- [ ] Kindly add a selection functionality on the transaction form, users should be able to choose a category they want to add a transaction to.

![trasact](https://github.com/MarwanDev/Tengo-Wallet/assets/53124548/2ec428d4-7f4e-4db9-bd24-cb8777e3f792)


![form](https://github.com/MarwanDev/Tengo-Wallet/assets/53124548/55abc154-3b32-4e40-8d34-23849f4402fc)